TOC-tidb-cloud.md

```diff
@@ -458,7 +459,31 @@
         - [TiKV Overview](/tikv-overview.md)
         - [RocksDB Overview](/storage-engine/rocksdb-overview.md)
     - TiFlash
-        - [TiFlash Overview](/tiflash/tiflash-overview.md)
+      - [TiFlash Overview](/tiflash/tiflash-overview.md)
+  - [CLI](/tidb-cloud/cli/cli-reference.md)
```

Comment:
The doc website does not support adding a file to a doc node. Please add a document "CLI reference" or "CLI overview" under CLI instead.